src/Admin/Actions.php

```diff
@@ -46,7 +46,19 @@ public function register_assets() {
 	}
 
 	public function save_admin_settings() {
-		$post_data = $_POST;
+		//$post_data = $_POST;  
+        $post_data = array();
+		
+		$post_data['action'] = sanitize_text_field(trim($_POST['action']));
+		$post_data['roadblock'] = sanitize_text_field(trim($_POST['roadblock']));
```

Comment:
@Ankur0812 Can we create a helper function similar to `wc_clean` to handle this dynamically?

Reply:
Done